test/e2e/feature/feature.go

```diff
@@ -0,0 +1,132 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package feature contains pre-defined features used by test/e2e and/or
+// test/e2e_node.
```

Comment:
how do we keep this in sync with the `pkg/features`?

didn't we talked about using those values directly so tests and feature gates are in sync by code?

Reply:
We did, and [you said](https://github.com/kubernetes/kubernetes/pull/112894#discussion_r996424246):
> I think that we should not create this relation Feature <---> FeatureGate, those are independent things that some times be related, a FeatureGate doesn't have to be related to a Feature necessarily.

This file here is about features, not feature gates (= `pkg/features`), and thus intentionally not forced to be in sync.

Reply:
oh, you caught me there, I missed that detail, so we call Feature something like NetworkPolicy or Ingress, sorry about that, I stand on my previous position :)

Reply:
I agree, we should have that separation. But nonetheless it is confusing and we'll need very good guidance in the pending KEP and the framework about when to use `FeatureGate` and when `Feature`.

I also still wonder whether we can come up with a different name than `Feature`. It's just too close to `FeatureGate`.

Reply:
@SergeyKanzhelev you will be interested on this PR, since you started the discussion here https://github.com/kubernetes/enhancements/pull/3042/files#r988729747 :)